docs/fundamentals/code-analysis/categories.md

```diff
@@ -10,19 +10,23 @@ helpviewer_keywords:
+The following table shows the different code analysis rule categories and provides a link to the rules in each category. It also lists the configuration value to use in an EditorConfig file to [bulk-configure rule severity](configuration-options.md#severity-level) on a per-category basis. For example, to set the severity of security rule violations to be errors, the EditorConfig entry is `dotnet_analyzer_diagnostic.category-Security.severity = error`.
 
-| Category | Description |
-| - | - |
-| [Design rules](quality-rules/design-warnings.md) | Design rules support adherence to the [.NET Framework Design Guidelines](../../standard/design-guidelines/index.md). |
-| [Documentation rules](quality-rules/documentation-warnings.md) | Documentation rules support writing well-documented libraries through the correct use of XML documentation comments for externally visible APIs. |
-| [Globalization rules](quality-rules/globalization-warnings.md) | Globalization rules support world-ready libraries and applications. |
-| [Portability and interoperability rules](quality-rules/interoperability-warnings.md) | Portability rules support portability across different platforms. Interoperability rules support interaction with COM clients. |
-| [Maintainability rules](quality-rules/maintainability-warnings.md) | Maintainability rules support library and application maintenance. |
-| [Naming rules](quality-rules/naming-warnings.md) | Naming rules support adherence to the naming conventions of the .NET design guidelines. |
-| [Performance rules](quality-rules/performance-warnings.md) | Performance rules support high-performance libraries and applications. |
-| [SingleFile rules](quality-rules/singlefile-warnings.md) | Single file rules support single-file applications. |
-| [Reliability rules](quality-rules/reliability-warnings.md) | Reliability rules support library and application reliability, such as correct memory and thread usage. |
-| [Security rules](quality-rules/security-warnings.md) | Security rules support safer libraries and applications. These rules help prevent security flaws in your program. |
-| [Style rules](style-rules/index.md) | Style rules support consistent code style in your codebase. These rules start with the "IDE" prefix. |
-| [Usage rules](quality-rules/usage-warnings.md) | Usage rules support proper usage of .NET. |
+> [!TIP]
+> Setting the severity for a category of rules using the `dotnet_analyzer_diagnostic.category-<category>.severity` syntax doesn't apply to rules that are disabled by default. However, starting in .NET 6, you can use the [AnalysisMode\<Category>](../../core/project-sdk/msbuild-props.md#analysismodecategory) project property to enable all the rules in a category.
+
+| Category | Description | EditorConfig value |
+| - | - | - |
+| Code quality rules | This category encompasses all rules except code-style rules of the form IDEXXXX. | `dotnet_analyzer_diagnostic.category-CodeQuality.severity` |
```

Comment:
This is not true - `category-CodeQuality` would not apply to any CA rule, one would need to specify individual CA category such as `category-Design`, `category-Security`, etc. I think we should just remove this row.

Reply:
@mavasani I believe this is not referring to CAxxxx. It's referring to the few IDExxxx CodeQuality rules in dotnet/roslyn

Reply:
Ah got it - I think we should just port those handful of IDExxxx code quality rules to analyzers repo and give them a CA ID and deprecate the corresponding IDE ID. Its a lot of work for not much value add though, so I am fine if we keep it the way it is.